src/pages/task/ActionButtons.tsx

```diff
@@ -271,7 +274,9 @@ export const ActionButtons: React.FC<Props> = ({
               onClick={() => {
                 taskAnalytics.sendEvent({ name: "Click See History Button" });
               }}
-              to={getTaskHistoryRoute(projectIdentifier, taskName)}
+              to={getTaskHistoryRoute(project.identifier, displayName, {
+                selectedCommit: order,
+              })}
```

Comment:
Would you mind double checking this? It's not working for me, for some reason the `order` is `undefined`. I might be doing something wrong though 🥲